.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+name: UCX-Py documentation build
+
+on:
+  push:
+    branches: ["branch-0.20"]
```

Comment:
Unfortunately, GitHub Actions does not have a way to dynamically select the default branch of a repo, so we would need to change this with every new release cycle.

Reply:
For this, you could adapt the lines of code below from `cudf` for `ucx-py`. That script gets run when we create new development branches and the `sed` commands can update this line for you automatically.

https://github.com/rapidsai/cudf/blob/branch-0.20/ci/release/update-version.sh#L44-L75